evaluator/perception_online_evaluator/include/perception_online_evaluator/perception_online_evaluator_node.hpp

```diff
@@ -61,6 +61,8 @@ class PerceptionOnlineEvaluatorNode : public rclcpp::Node
 
   DiagnosticStatus generateDiagnosticStatus(
     const std::string metric, const Stat<double> & metric_stat) const;
+  DiagnosticStatus generateDiagnosticStatus(
+    const std::string metric, const double metric_value) const;
```

Comment:
[NITS] How about using const reference?  
```suggestion
    const std::string & metric, const double metric_value) const;
```